### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by FilippoBonazziSUSE** [openSUSE/openSUSEway#89] [Polls]
As we started discussing in #88, it's been a while since the last release.

We have a few points open for improvement still, but also several features that have been merged in the past year and never released.

Would you want to wait to improve more before a new release, or start with a new release and iterate on that?

**Comment by denisok** [comment]
@FilippoBonazziSUSE added 0.15-rc, updating package and testing it. I also added you as a maintainer - feel free to roll next releases when you feel like it :)
